mysql/datadog_checks/mysql/mysql.py

```diff
@@ -535,23 +535,23 @@ def _collect_metrics(self, db, tags):
             # report size of tables in MiB to Datadog
             self.log.debug("Collecting Table Row Stats Metrics.")
             with tracked_query(self, operation="table_rows_stats_metrics"):
-                (rows_read_total, rows_changed_total) = self._query_rows_stats_per_table(db)
+                (rows_read_total, rows_changed_total) = self._query_rows_stats_per_table(db) or (None, None)
```

Comment:
how about fix the return value https://github.com/DataDog/integrations-core/blob/34ac7eec122aabe0522f7fa9f23431dc984701b3/mysql/datadog_checks/mysql/mysql.py#L1220 and https://github.com/DataDog/integrations-core/blob/34ac7eec122aabe0522f7fa9f23431dc984701b3/mysql/datadog_checks/mysql/mysql.py#L1237? this way the tuple can be properly unpacked.